pkg/version/version_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetFullVersion(t *testing.T) {
+	gotversion := GetFullVersion()
+	assert.Equal(t, "UNKNOWN", gotversion)
+
+	Version = "1.9"
+	ReleaseStatus = "released"
+	gotversion = GetFullVersion()
```

Comment:
Ditto, better to rename `gotversion` to `gotVersion` with camel case.

Reply:
+1